docs/4.4. Behaviour - Access Tokens.md

```diff
@@ -108,11 +111,15 @@ token is used for.
 ### x-nmos-*
 _Contains information particular to the NMOS API the token is intended for_
 
-One or more `x-nmos-*` claims MUST be included in the token. The claim name begins `x-nmos-` followed by the namespace
+One or more `x-nmos-*` claims SHOULD be included in the token. The claim name begins `x-nmos-` followed by the namespace
 identifier found in the URL for that given API (e.g. "registration", "query", etc.).
 
-The value of the claim is a JSON object, indicating access permissions for that API. An omitted `x-nmos-*` object indicates
-that no access is permitted to the namespace-identified API.
+Presence of an `x-nmos-*` claim matching an NMOS API grants implicit read only access to some API base paths as
+specified in [Resource Servers](4.5.%20Behaviour%20-%20Resource%20Servers.md).
```

Comment:
Why this as well as the "Presence of a `scope` matching an NMOS API grants..." statement?

Reply:
The `scope` is a SHOULD, so I was covering the case where that may be omitted.

Reply:
I've just noticed that... Why is the `scope` only recommended? I'd assumed it was a more fundamental check before moving on to the `x-nmos-*` claims.

Reply:
I believe (but would have to check) that this follows what the OAuth RFCs require, so we don't mandate `scope` in case a chosen Auth Server implementation doesn't include it in tokens.

Reply:
Ah OK. Hence the hoops to allow one or other or both for the read-access to API base paths. OK.